src/main/java/seedu/weme/logic/commands/importcommand/LoadCommand.java

```diff
@@ -31,6 +31,8 @@
             + PREFIX_FILEPATH + "C:/Users/username/Downloads/ ";
     public static final String MESSAGE_SUCCESS = "Memes loaded successfully to the import staging area.";
     public static final String MESSAGE_LOAD_FAILURE = "Invalid directory path given.";
+    public static final String MESSAGE_LOAD_NON_EMPTY_TAB_FAILURE = "There are memes present in the import tab. "
```

Comment:
Hmm why?

Reply:
You can't do load twice because exception is not handled. I just want to handle the exception.

Reply:
I am using UniqueMemeList for importlist. Loading twice from same directory will result in duplicate memes